rebar.config

```diff
@@ -5,3 +5,19 @@
 {profiles, [
     {test, [{erl_opts, [nowarn_export_all]}]}
 ]}.
+
+%% the plugin itself
+{plugins, [rebar3_proper]}.
```

Comment:
use `project_plugins` so we won't fetch the plugins as a transitive dep

Reply:
ok